docs/CommandLineOptions.html

```diff
@@ -250,7 +250,10 @@ <h2>Options allowed only on the command line</h2>
 
 
 <dt><code>filterReads</code>
-<dd>Shasta performs read loading and filtering, prints the passing read names to a CSV, and exits.</dd>
+<dd>
+Shasta performs read loading and filtering, prints the passing read names to a CSV, and exits.
+If palindromes are detected, their read names are written separately to PalindromicReads.csv
```

Comment:
Please enclose the file name in a `code` tag (like `filterReads` 3 lines above).